### PR TITLE
Special Two Digit Number

### PR DESCRIPTION
**Pull request by Arghadip-Chatterjee**
A special two-digit number is a number such that when the sum of the digits of the number is added to the product of its digits, the result is equal to the original two-digit number.

### CODE REVIEW

File: src/main/java/org/example/basics/special_twodigit_number.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import java.util.*;
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/basics/special_twodigit_number.java:1]: package info is missing at top of file as 
```java
package org.example.basics;
```

**Review by BimalRajGyawali — CHANGES_REQUESTED** [review]

**Comment by Arghadip-Chatterjee** [comment]
Hi Sir. Can you please explain what is package info and how to add it? I
have used the scanner class which itself is a package. I am getting
confused. Please guide me. What is the meaning of this line

package org.example.basics;

**Comment by BimalRajGyawali** [comment]
Hi @Arghadip-Chatterjee, at top of every class there should be package information indicating the package the class belongs to.
You can see other classes in this repo for reference.

**Comment by Arghadip-Chatterjee** [comment]
Sir I have made the changes and again uploaded the file in the repository.
Please merge it now. Thanks sir for the clarification.

**Pull request closed without merging**